inc/3rd-party/themes/uncode.php

```diff
@@ -59,8 +59,31 @@ function rocket_exclude_defer_js_uncode( $exclude_defer_js ) {
 		$exclude_defer_js[] = rocket_clean_exclude_file( get_template_directory_uri() . '/library/js/init.js' );
 		$exclude_defer_js[] = rocket_clean_exclude_file( get_template_directory_uri() . '/library/js/min/init.min.js' );
 		$exclude_defer_js[] = rocket_clean_exclude_file( get_template_directory_uri() . '/library/js/init.min.js' );
+		$exclude_defer_js[] = rocket_clean_exclude_file( get_template_directory_uri() . '/library/js/min/ai-uncode.min.js' );
+		$exclude_defer_js[] = rocket_clean_exclude_file( get_template_directory_uri() . '/library/js/ai-uncode.min.js' );
+		$exclude_defer_js[] = rocket_clean_exclude_file( get_template_directory_uri() . '/library/js/ai-uncode.js' );
 		return $exclude_defer_js;
 	}
 	add_filter( 'rocket_exclude_defer_js', 'rocket_exclude_defer_js_uncode' );
 
+	/**
+	 * Excludes Uncode JS files from delay JS
+	 *
+	 * @since 3.10.5
+	 *
+	 * @param array $exclude_delay_js Array of JS to be excluded.
+	 * @return array
+	 */
+	function rocket_exclude_delay_js_uncode( $exclude_delay_js ) {
+		$exclude_delay_js[] = rocket_clean_exclude_file( get_template_directory_uri() . '/library/js/init.js' );
+		$exclude_delay_js[] = rocket_clean_exclude_file( get_template_directory_uri() . '/library/js/min/init.min.js' );
+		$exclude_delay_js[] = rocket_clean_exclude_file( get_template_directory_uri() . '/library/js/init.min.js' );
+		$exclude_delay_js[] = rocket_clean_exclude_file( get_template_directory_uri() . '/library/js/min/ai-uncode.min.js' );
+		$exclude_delay_js[] = rocket_clean_exclude_file( get_template_directory_uri() . '/library/js/ai-uncode.min.js' );
+		$exclude_delay_js[] = rocket_clean_exclude_file( get_template_directory_uri() . '/library/js/ai-uncode.js' );
+		$exclude_delay_js[] = 'UNCODE\.';
+		return $exclude_delay_js;
+	}
+	add_filter( 'exclude_delay_js', 'rocket_exclude_delay_js_uncode' );
```

Comment:
I think` exclude_delay_js` shall be `rocket_exclude_delay_js`  to follow the same naming convention as the rest. what do you think @mostafa-hisham?